envoy/network/connection.h

```diff
@@ -186,7 +193,7 @@ class Connection : public Event::DeferredDeletable,
    * readDisable(false);  // Notes the connection is blocked by one source
    * readDisable(false);  // Marks the connection as unblocked, so resumes reading.
    */
-  virtual void readDisable(bool disable) PURE;
+  virtual ReadDisableStatus readDisable(bool disable) PURE;
```

Comment:
Nit: please document the `@return` value

Reply:
done